samples/using-oidc-js-sdk/react-js-app/config/env.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
Shouldn't the sample files need the license headers as well?

Reply:
Fixed by 5c8fc5d